packages/frontend/src/components/table/RatingCell.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { renderToStaticMarkup } from 'react-dom/server'
+
+import { RatingBadge } from '../rating/Badge'
+import { RatingProps, RatingTooltipPopup } from '../rating/TooltipPopup'
+import { NoInfoCell } from './NoInfoCell'
+
+export function RatingCell({ item }: RatingProps) {
+  if (!item) {
+    return <NoInfoCell />
+  }
+
+  return (
+    <div
+      className="Tooltip"
+      title={renderToStaticMarkup(<RatingTooltipPopup item={item} />)}
```

Comment:
Interesting idea.

Alternative suggestion: Maybe we can instead have something like `data-tooltip-content="some-id"` and then we have this content hidden as a child of `RatingCell` and we use this `.clone()` trick from earlier to replace tooltip content?